test/unit/spec/common/cellComponents/fieldTableCellWidget-Spec.js

```diff
@@ -127,7 +127,7 @@ define([
                     );
                 });
             }
-            this.parent.innerHTML = '';
+            $(this.parent).remove();
```

Comment:
remove the node instead of emptying it

Reply:
`.remove()` is part of the DOM api now, so jquery isn't necessary.
https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove

Reply:
I will get rid of jQuery, in that case!